src/Xamarin.Android.Build.Tasks/Tasks/GenerateJniRemappingNativeCode.cs

```diff
@@ -152,7 +152,7 @@ bool GetRequiredAttribute (string attributeName, out string attributeValue)
 					return true;
 				}
 
-				Log.LogError ($"Attribute '{attributeName}' missing from element '{reader.LocalName}'; {RemappingXmlFilePath.ItemSpec} line {GetCurrentLineNumber ()}");
+				Log.LogError ($"Attribute '{attributeName}' missing from element '{reader.LocalName}'; {RemappingXmlFilePath!.ItemSpec} line {GetCurrentLineNumber ()}");
```

Comment:
Instead of using `!` here, can we pass in a non-nullable `string remappingXmlFilePath`.